test/Microsoft.Azure.ServiceBus.UnitTests/API/ApiApprovals.ApproveAzureServiceBus.approved.txt

```diff
@@ -423,6 +423,15 @@ namespace Microsoft.Azure.ServiceBus.Core
         public virtual System.Threading.Tasks.Task<Microsoft.Azure.ServiceBus.Message> BeforeMessageSend(Microsoft.Azure.ServiceBus.Message message) { }
     }
 }
+namespace Microsoft.Azure.ServiceBus.Extensions
+{
+    
+    public class static MessageInteropExtensions
+    {
+        public static T GetBody<T>(this Microsoft.Azure.ServiceBus.Message message) { }
+        public static T GetBody<T>(this Microsoft.Azure.ServiceBus.Message message, System.Runtime.Serialization.XmlObjectSerializer serializer) { }
```

Comment:
Once comment from @binzywu is addressed, this will need to be re-approved to remove this overload.